mkdocs.yml

```diff
@@ -1,20 +1,95 @@
 site_name: Guidelines and Documentation
 pages:
 - Home: 'index.md'
+- Platform Features:
+  - Structure:
+    - 'Courses': 'features/structure/courses.md'
+    - 'Sections': 'features/structure/sections.md'
+    - 'Items': 'features/structure/items.md'
+  - Item Types:
+    - 'Video': 'features/itemtypes/video.md'
+    - 'Quiz': 'features/itemtypes/quiz.md'
+    - 'Surveys': 'features/itemtypes/surveys.md'
+    - 'Text': 'features/itemtypes/text.md'
+    - 'LTI': 'features/itemtypes/lti.md'
+    - 'Peer Assessment': 'features/itemtypes/peerassessment.md'
+   - Discussion Forum: 'features/discussionforum.md'
+   - Collab spaces: 'features/collabspaces.md'
+   - Gamification: 'features/gamification.md'
+  - Course management:
+    - 'Student Enrollments': 'features/coursemanagement/studentenrollments.md'
+    - 'Permissions': 'features/coursemanagement/permissions.md'
+    - 'Pinboard abuse reports': 'features/coursemanagement/pinboardabusereports.md'
+    - 'Student Submissions': 'features/coursemanagement/studentsubmissions.md'
+  - Analytics:
+    - Dashboards:
+      - 'Item Visits': 'features/analytics/dashboards/itemvisits.md'
+      - 'Videos': 'features/analytics/dashboards/videos.md'
+      - 'Downloads': 'features/analytics/dashboards/downloads.md'
+      - 'Quizzes': 'features/analytics/dashboards/quizzes.md'
+      - 'Item Details': 'features/analytics/dashboards/itemdetails.md'
+      - 'Section Conversions': 'features/analytics/dashboards/sectionconversions.md'
+      - 'Forum': 'features/analytics/dashboards/forum.md'
+      - 'Activity': 'features/analytics/dashboards/activity.md'
+      - 'Geographic': 'features/analytics/dashboards/geographic.md'
+      - 'Announcements': 'features/analytics/dashboards/announcements.md'
+      - 'QC Alerts': 'features/analytics/dashboards/qcalerts.md'
+      - 'Social Shares': 'features/analytics/dashboards/socialshares.md'
+      - 'Referrer': 'features/analytics/dashboards/referrer.md'
+    - Progress: 'features/analytics/progress.md'
+  - Certificates: 'features/certificates.md'
+  - Learning Objectives: 'features/learningobjectives.md'
+  - Course Announcements: 'features/courseannouncements.md'
+  - External Tools: 
+    - CodeOcean: 'features/externaltools/codeocean.md'
+    - TeamBuilder: 'features/externaltools/teambuilder.md'
```

Comment:
Do we want to add some information on CodeOcean / TeamBuilder in our public Teaching Team Guidelines? If yes: How much?